test/e2e/framework/util/job/util.go

```diff
@@ -0,0 +1,25 @@
+package job
+
+import (
+	batch "k8s.io/api/batch/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newBool(val bool) *bool {
+	p := new(bool)
+	*p = val
+	return p
+}
+
+// FinishTime returns finish time of the specified job.
+func FinishTime(finishedJob *batch.Job) metav1.Time {
```

Comment:
I don't see this used anywhere, can we remove?

Reply:
The FinishTime is originaly called `JobFinishTime`, the golint warns me like this:

```go
func name will be used as job.JobFinishTime by other package, and that stutters; consider callling this FinishTime
```